4_arm.ipynb

```diff
@@ -371,13 +671,26 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "算出された時間軌道を実際に実行します。\n",
     "\n",
     "生成された軌道を引数として与えて以下を実行することで、アームが逆運動学の計算結果に従って動きます。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "We actually execute the calculated time trajectory.\n",
+    "\n",
+    "The arm moves according to the calculation result of inverse kinematics by giving the generated trajectory as an argument of `execute()` function:"
```

Comment:
"The arm moves according to the calculation result of inverse kinematics by giving the generated trajectory as an argument to the  `execute()` function:"